module-2/backend/week_two.md

```diff
@@ -26,20 +34,50 @@ end
 
 Now how would you find the owner of the team with an id of 4?
 
+Owner.where(team_id: 4)
+
 5. In a database that's holding students and teachers, what will be the relationship between students and teachers? Draw the schema diagram.
+Could be one to many or many to many depending on the type of school and grade level. 
+
+One to many: teacher will have many students, students belong to a teacher. Students have a teacher id along with the other attributes they have without the associations.
+
 6. Define foreign key, primary key, and schema.
+Foreign key of teacher id lives on student and the primary key would be that.
```

Comment:
`would be that`? Also, make sure you're answering the full question. Asked you to define schema as well.

I don't think this question is specifically related to the teacher example above, though it would be fine to reference it. We're looking for more general definitions.